static/client/common.js

```diff
@@ -169,7 +171,61 @@ CCC.Common.commandFunction = function() {
   if (CCC.Common.isConnected) {
     parent.postMessage({'commands': [this.innerText]}, location.origin);
   }
-  parent.focus();
+  CCC.Common.parentFocus();
+};
+
+/**
+ * The user pressed a key with the focus in the world/log frame.
+ * Move focus back to the parent frame and inject the keystroke into the
+ * command area.
+ * @param {!KeyboardEvent} e Keyboard down event.
+ */
+CCC.Common.keyDown = function(e) {
+  if (e.key === 'Alt' || e.key === 'Control' || e.key === 'Meta') {
+    // Don't steal focus if the user is pressing a modifier key in preparation
+    // for a cut/copy operation.
+    return;
+  }
+  if (e.ctrlKey || e.altKey || e.metaKey) {
+    // Allow Chrome time to complete a copy before moving focus.
+    setTimeout(CCC.Common.parentFocus, 0);
```

Comment:
Should `e` be a parameter to `parentFocus` here?  I am thinking in particular of the case that the user is cursoring with Emacs cursor keys (C-f, C-b, C-n, C-p etc.)

Reply:
Tricky.  That means that on Windows C-c, C-x, apparently need to be handled differently than C-f, etc.  Let's revisit this issue when we are together, otherwise I'd be adding untested code and guessing at the results.